batch/batch/worker/worker.py

```diff
@@ -132,6 +135,10 @@ def compose(auth: Union[MutableMapping, str, bytes], registry_addr: str = None):
 INTERNET_INTERFACE = os.environ['INTERNET_INTERFACE']
 UNRESERVED_WORKER_DATA_DISK_SIZE_GB = int(os.environ['UNRESERVED_WORKER_DATA_DISK_SIZE_GB'])
 assert UNRESERVED_WORKER_DATA_DISK_SIZE_GB >= 0
+ACCEPTABLE_QUERY_JAR_URL_PREFIX = os.environ['ACCEPTABLE_QUERY_JAR_URL_PREFIX']
+assert (
```

Comment:
This doesn't seem like a very strong assert to me. The environment variable should always be a string.